cmake/SSGCommon.cmake

```diff
@@ -1137,7 +1137,7 @@ macro(ssg_build_html_stig_tables PRODUCT)
     add_custom_command(
         OUTPUT "${CMAKE_BINARY_DIR}/tables/table-${PRODUCT}-stig-manual.html"
         COMMAND "${CMAKE_COMMAND}" -E make_directory "${CMAKE_BINARY_DIR}/tables"
-        COMMAND "${XSLTPROC_EXECUTABLE}" --output "${CMAKE_BINARY_DIR}/tables/table-${PRODUCT}-stig-manual.html" "${CMAKE_CURRENT_SOURCE_DIR}/transforms/xccdf2table-stig.xslt" "${DISA_STIG_REF}"
+        COMMAND "${XSLTPROC_EXECUTABLE}" --output "${CMAKE_BINARY_DIR}/tables/table-${PRODUCT}-stig-manual.html" "${CMAKE_CURRENT_SOURCE_DIR}/transforms/xccdf2table-stig-manual.xslt" "${DISA_STIG_REF}"
```

Comment:
@jan-cerny What if we add a `--string-param`  for the XCCDF namespace instead of duplicating the XSLT?

Reply:
Hmm, actually, I guess they don't work on the namespace....

Reply:
That would be great. Do you have any idea how to do that? The namespace influences / is a part of the select and match attributes and XPath expressions. Is there a way how to parametrize those by namespace?